test/integration/agent/testdata/fleet.yaml

```diff
@@ -11,8 +11,6 @@ spec:
       labels:
         fleet: default
     spec:
-      os:
-        image: quay.io/redhat/rhde:9.2
```

Comment:
note: this was removed because integration tests should not perform an upgrade because the upgrade would be targeting the host.